istioctl/cmd/proxyconfig.go

```diff
@@ -762,22 +762,31 @@ func logCmd() *cobra.Command {
 			}
 
 			var resp string
-			if len(destLoggerLevels) == 0 {
-				resp, err = setupEnvoyLogConfig("", podName, podNamespace)
-			} else {
-				if ll, ok := destLoggerLevels[defaultLoggerName]; ok {
-					// update levels of all loggers first
-					resp, err = setupEnvoyLogConfig(defaultLoggerName+"="+levelToString[ll], podName, podNamespace)
-					delete(destLoggerLevels, defaultLoggerName)
+			var errs *multierror.Error
+			for _, podName := range podNames {
+				if len(destLoggerLevels) == 0 {
+					resp, err = setupEnvoyLogConfig("", podName, podNamespace)
+				} else {
+					if ll, ok := destLoggerLevels[defaultLoggerName]; ok {
+						// update levels of all loggers first
+						resp, err = setupEnvoyLogConfig(defaultLoggerName+"="+levelToString[ll], podName, podNamespace)
+					}
+					for lg, ll := range destLoggerLevels {
+						if lg == defaultLoggerName {
+							continue
+						}
+						resp, err = setupEnvoyLogConfig(lg+"="+levelToString[ll], podName, podNamespace)
+					}
 				}
-				for lg, ll := range destLoggerLevels {
-					resp, err = setupEnvoyLogConfig(lg+"="+levelToString[ll], podName, podNamespace)
+				if err != nil {
+					errs = multierror.Append(errs, fmt.Errorf("error configuring log level for %v.%v: %v", podName, podNamespace, err))
+				} else {
+					_, _ = fmt.Fprint(c.OutOrStdout(), resp)
```

Comment:
Also include the pod name before printing its log levels?

Reply:
done